scanpy/readwrite.py

```diff
@@ -277,15 +277,53 @@ def _read_v3_10x_h5(filename, *, start=None):
                 (data, dsets['indices'], dsets['indptr']),
                 shape=(N, M),
             )
+            obs_dict = dict(obs_names=dsets['barcodes'].astype(str))
+            var_dict = dict(
+                var_names=dsets['name'].astype(str),
+            )
+
+            # Handles reading feature-barcode matrix
+            if 'gene_id' not in dsets:
```

Comment:
```suggestion
            if 'gene_id' not in dsets:
                # Read the feature-barcode matrix metadata.
```
I prefer to put the comment inside the block and use the imperative tense (e.g. `Read` rather than `Reads`). Ditto the comment below.

Reply:
Fixed.